services/leaf_space_integration/contact_reservations/parsers.go

```diff
@@ -1,27 +1,27 @@
 package leaf_space_contact_reservations
```

Comment:
package and folder name should be changed accordingly

Reply:
Ok done